gradle-plugin/README.md

```diff
@@ -210,20 +210,26 @@ emerge {
   ..
 
   snapshots {
+    buildType.set("snapshots") // Build type to use for grouping builds in the Emerge dashboard
+
+    fromMainSourceSet.set(
+      true
+    ) // Generate composable previews snapshots from the main sourceSet, defaults to `false`
+
     snapshotsStorageDirectory.set(
       "/src/main/snapshots"
     ) // Path to local snapshot image storage, defaults to `/build/emerge/snapshots/outputs`
-    buildType.set("snapshots") // Build type to use for grouping builds in the Emerge dashboard
```

Comment:
Why remove?

Reply:
Didn't remove, just moved up to match the ordering in the table below this.